webapp/controller/ListReport.controller.js

```diff
@@ -74,61 +75,14 @@ sap.ui.define([
 		onRouteMatched: function() {
 			const oTable = this.byId("idProductsTable");
 			const oItemsBinding = oTable.getBinding("items");
-			const oSorter = new Sorter("CreatedAt", "ASC");
-			
-			oItemsBinding.sort(oSorter);
-		},
-
-		getPriceFilter: function() {
-			const oPriceModel = this.getModel("PriceModel");
-			const sSelectedKey = oPriceModel.getProperty("/SelectedKey");
-			const aPricesRange = oPriceModel.getProperty("/Range");
-			const [oCurrentPriceRange] = aPricesRange.filter(el => el.id === sSelectedKey);
-			const sRegNumbers = /[\wa-zA-Z]+|\d+/g;
-			const aCurrentPriceRange = oCurrentPriceRange.value.match(sRegNumbers);
-			const oWords = {
-				any: this.getTextFromI18n("AnyText"),
-				under: this.getTextFromI18n("UnderText"),
-				over: this.getTextFromI18n("OverText")
-			}
-			
-			const [sFirstRange, sSecondRange] = aCurrentPriceRange;
-			switch(sFirstRange) {
-				case oWords.any: return [new Filter("Price", FilterOperator.NE, null)]; 
-				case oWords.under: return [new Filter("Price", FilterOperator.BT, 0, Number(sSecondRange))];
-				case oWords.over: return [new Filter("Price", FilterOperator.GT, Number(sSecondRange))];
-				default: return [new Filter("Price", FilterOperator.BT, Number(sFirstRange), Number(sSecondRange))];
-			}
-		},
-
-		handlerTokenUpdate: function(oEvent) {
-			const oMultiInput = this.byId("multiInput");
+			console.log(oItemsBinding)
+			const oSorter = new Sorter("ReleaseDate", "ASC");
 			
-			if (oEvent?.getParameter("type") === "removed") {
-				const oModel = this.getModel();
-				const aSuppliers = oModel.getProperty("/Suppliers");		
-				const aRemovedTokens = oEvent.getParameter("removedTokens");
-				const aRemainingTokens = oMultiInput.getTokens().filter(function (token) {
-					return !aRemovedTokens.includes(token);
-				});
-				
-				const aSelectedTokens = aRemainingTokens.map(el => el.getText());
-				const aSelectedId = aSuppliers
-										.filter((el) => aSelectedTokens.includes(el.SuppliersName))
-										.map((el) => el.SupplierId);
-				
-				this.onFilter(aSelectedId);	
-			}
+			oItemsBinding.sort(oSorter);	
```

Comment:
You can do it in xml view the same way you provide values for expand query option.
It's the best practice to provide static query options in xml view, in js provide only dynamic query options